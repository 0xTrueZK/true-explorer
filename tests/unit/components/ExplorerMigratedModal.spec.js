import MockHelper from '../MockHelper';

import ExplorerMigratedModal from '@/components/ExplorerMigratedModal.vue';

let helper;
beforeEach(() => {
    jest.clearAllMocks()
    helper = new MockHelper();
});

describe('ExplorerMigratedModal.vue', () => {
    it('Should display waiting screen', async () => {
        const wrapper = helper.mountFn(ExplorerMigratedModal, {
            data() {
                return {
                    dialog: true,
                    explorerId: 1
                }
            }
        });

        expect(wrapper.html()).toMatchSnapshot();
    });
});
