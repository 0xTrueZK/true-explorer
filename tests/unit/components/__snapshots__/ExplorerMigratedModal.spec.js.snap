// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExplorerMigratedModal.vue Should display waiting screen 1`] = `
<div class="v-dialog__container">
  <div role="dialog" aria-modal="true" class="v-dialog__content v-dialog__content--active" style="z-index: 202;">
    <transition-stub name="dialog-transition" appear="true">
      <div tabindex="0" class="v-dialog v-dialog--active v-dialog--persistent" style="transform-origin: center center; max-width: 800px;">
        <div class="v-card v-sheet v-sheet--outlined theme--light">
          <div class="v-card__text mt-4">
            <div role="progressbar" aria-valuemin="0" aria-valuemax="100" class="v-progress-circular mr-2 v-progress-circular--visible v-progress-circular--indeterminate primary--text" style="height: 16px; width: 16px;"><svg xmlns="http://www.w3.org/2000/svg" viewBox="22.857142857142858 22.857142857142858 45.714285714285715 45.714285714285715" style="transform: rotate(0deg);">
                <circle fill="transparent" cx="45.714285714285715" cy="45.714285714285715" r="20" stroke-width="5.714285714285714" stroke-dasharray="125.664" stroke-dashoffset="125.66370614359172px" class="v-progress-circular__overlay"></circle>
              </svg>
              <div class="v-progress-circular__info"></div>
            </div>
            Your explorer is being finalized. You will be redirected automatically as soon as it's ready.
          </div>
        </div>
      </div>
    </transition-stub>
  </div>
</div>
`;
